src/Draco.Editor.Web/app/ts/app.ts

```diff
@@ -0,0 +1,120 @@
+import * as monaco from 'monaco-editor';
```

Comment:
Why does the file randomly omit semicolons? I know that JS defines weird semicolon omission rules but do we have to make use of that?

Reply:
Can we get some minimal doc commenting in this file? In case anyone else has to work on it who is not well-seasoned in the JS/TS ecosystem